src/analysis/inner_product_space/basic.lean

```diff
@@ -1570,19 +1583,6 @@ Compare `abs_inner_eq_norm_iff`, which takes the weaker hypothesis `abs ⟪x, y
 lemma inner_eq_norm_mul_iff_real {x y : F} : ⟪x, y⟫_ℝ = ∥x∥ * ∥y∥ ↔ ∥y∥ • x = ∥x∥ • y :=
 inner_eq_norm_mul_iff
 
-/-- An inner product space is strictly convex. We do not register this as an instance for an inner
-space over `𝕜`, `is_R_or_C 𝕜`, because there is no order of the typeclass argument that does not
-lead to a search of `[is_scalar_tower ℝ ?m E]` with unknown `?m`. -/
-instance inner_product_space.strict_convex_space : strict_convex_space ℝ F :=
-begin
-  refine strict_convex_space.of_norm_add (λ x y h, _),
-  rw [same_ray_iff_norm_smul_eq, eq_comm, ← inner_eq_norm_mul_iff_real,
-    real_inner_eq_norm_add_mul_self_sub_norm_mul_self_sub_norm_mul_self_div_two, h,
-    add_mul_self_eq, sub_sub, add_sub_add_right_eq_sub, add_sub_cancel', mul_assoc,
-    mul_div_cancel_left],
-  exact _root_.two_ne_zero
-end
```

Comment:
@urkud, do you wish to keep this proof around? Would it work to prove `inner_product_space ℂ E → strict_convex_space ℂ E`?

Reply:
I don't want to talk about `strict_convex_space ℂ E`, so let's remove this proof. BTW, I have a better version of the lemma added in #13548 (I drop assumptions `0 < a` and `0 < b`; also, the proof is based on a more general fact about strict convex sets). It depends on #13631, #13656, and #13658.

Reply:
Oh exciting! Ping me when it's out.

Reply:
Let's merge your PR first.